Production_scripts/Process_extracts/Produce_extract_tests/extract_nrs_tests.R

```diff
@@ -0,0 +1,38 @@
+####################################################
+# Name of file - extract_nrs_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               acute processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+
+# Read in Data-----------------------------------------
+
+year <- check_year_format("1920")
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Deaths"))
+
+# Read current SLF episode file and filter for 01B and GLS records
+existing_data <- get_existing_data_for_tests(new_data = new_data)
+
+
+# Produce comparison-------------------------------------
+# Compare new file with existing slf data
+comparison <- produce_test_comparison(
+  produce_nrs_extract_tests(existing_data),
+  produce_nrs_extract_tests(new_data)
```

Comment:
```suggestion
  produce_source_nrs_tests(existing_data),
  produce_source_nrs_tests(new_data)
```
suggestion to change the function name in line with other tests. Especially PIS tests which dont follow the other 'typical' tests. i.e. it's own test function

Reply:
Changed function name